src/filter.js

```diff
@@ -42,7 +42,21 @@ const EXCLUDED_MATERIAL_TYPES = [
 
 export default function (record, earliestCatalogTime) {
 	const leader = record.varFields.find(f => f.fieldTag === '_');
-	const materialType = record.materialType.code.trim();
+
+	const materialType = function () {
+		if (record.materialType === undefined) {
+			return false;
+		}
```

Comment:
This block is superfluous since that property is tested for the next block and there's no explicit `return` the function will return `undefined` which then will make the code to enter the block `if (!materialType) {`

Reply:
Modifications done and run lint.